### PR TITLE
OSDOCS-6922: Infrastructure Node Utilization Update

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.13+`

Issue:
[OSD-15310](https://issues.redhat.com/browse/OSD-15310)

Link to docs preview:
* [Node scaling after installation](https://63960--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-limits-scalability.html#node-scaling-after-installation_rosa-limits-scalability) in Limits and Scalability
    image

QE review:
- [ ] QE has approved this change.

Additional information:
Updated the infrastructure node scaling to specify high-sustained utilization.

### CODE REVIEW

**Comment by EricPonvelle** [comment]
@mjlshen Can you tell me if this is what you envisioned for these changes? Thanks!

**Comment by mjlshen** [comment]
cc @clcollins

File: modules/sd-planning-considerations.adoc [hunk 1]
```diff
@@ -70,10 +70,10 @@ The maximum number of compute nodes on ROSA is 180.
 
 .Rules for infrastructure node resizing alerts
 
-Resizing alerts are triggered for the infrastructure nodes in a cluster when either of the following scenarios are true: 
+Resizing alerts are triggered for the infrastructure nodes in a cluster when the CPU or the memory has high sustained utilization. This high-sustained utilization status is: 
 
-* Each infrastructure node has less than 16GiB RAM or less than 5 CPUs, and there are more than 25 and less than 101 compute nodes.
-* Each infrastructure node has less than 32GiB RAM or less than 9 CPUs, and there are more than 100 compute nodes.
+* An infrastructure node sustains over 50% utilization in a classic ROSA cluster with a single availability zones using 2 infrastructure nodes.
```
Comment (mjlshen) [inline modules/sd-planning-considerations.adoc:75]:
```suggestion
* Infrastructure nodes sustain over 50% utilization on average in a classic ROSA cluster with a single availability zones using 2 infrastructure nodes.
```

The alert triggers off of the total average (e.g. one node at 60% and one node at 30% wouldn't trigger a scaleup, but both nodes at 60% would) - open to other ways to word this too.

File: modules/sd-planning-considerations.adoc [hunk 2]
```diff
@@ -70,10 +70,10 @@ The maximum number of compute nodes on ROSA is 180.
...
+* An infrastructure node sustains over 66% utilization in a classic ROSA cluster with with multiple availability zones using 3 infrastructure nodes.
```
Comment (mjlshen) [inline modules/sd-planning-considerations.adoc:76]: Whatever we update the previous line's wording to should be mirrored here

**Review by mjlshen — CHANGES_REQUESTED** [review]
Some feedback on the details, overall looks good!

**Review by cbippley — COMMENTED** [review]
You've got a double "with" on line 76. Other than that, LGTM!

**Comment by cbippley** [comment]
/label peer-review-done
/remove-label peer-review-in-progress
As I'm in training, please wait for @opayne1 to take a look as well before requesting merge review.

File: modules/sd-planning-considerations.adoc [hunk 3]
```diff
@@ -70,10 +70,10 @@ The maximum number of compute nodes on ROSA is 180.
 
 .Rules for infrastructure node resizing alerts
 
-Resizing alerts are triggered for the infrastructure nodes in a cluster when either of the following scenarios are true: 
+Resizing alerts are triggered for the infrastructure nodes in a cluster when the CPU or the memory has high sustained utilization. This high-sustained utilization status is: 
 
-* Each infrastructure node has less than 16GiB RAM or less than 5 CPUs, and there are more than 25 and less than 101 compute nodes.
-* Each infrastructure node has less than 32GiB RAM or less than 9 CPUs, and there are more than 100 compute nodes.
+* Infrastructure nodes sustain over 50% utilization on average in a classic ROSA cluster with a single availability zones using 2 infrastructure nodes.
+* Infrastructure nodes sustain over 66% utilization on average in a classic ROSA cluster with with multiple availability zones using 3 infrastructure nodes.
```
Comment (cbippley) [inline modules/sd-planning-considerations.adoc:76]:
```suggestion
* Infrastructure nodes sustain over 66% utilization on average in a classic ROSA cluster with multiple availability zones using 3 infrastructure nodes.
```

File: modules/sd-planning-considerations.adoc [hunk 4]
```diff
@@ -70,10 +70,10 @@ The maximum number of compute nodes on ROSA is 180.
 
 .Rules for infrastructure node resizing alerts
 
-Resizing alerts are triggered for the infrastructure nodes in a cluster when either of the following scenarios are true: 
+Resizing alerts are triggered for the infrastructure nodes in a cluster when the CPU or the memory has high sustained utilization. This high-sustained utilization status is:
```
Comment (mjlshen) [inline modules/sd-planning-considerations.adoc:73]:
```suggestion
Resizing alerts are triggered for the infrastructure nodes in a cluster when it has high-sustained CPU or memory utilization. This high-sustained utilization status is: 
```

Standardizing the hyphenated `high-sustained` and removing the `the CPU`/`the memory` wording

**Review by mjlshen — APPROVED** [review]
/lgtm

Thanks Eric!

**Review by mjlshen — APPROVED** [review]
/lgtm

**Pull request merged**